.prettierignore

```diff
@@ -1,3 +1,3 @@
 dist
 .nuxt
-.github
\ No newline at end of file
+pnpm-lock.yaml
```

Comment:
`pnpm-lock.yaml` is generated automatically by `pnpm`, and its formatting isn't prettier-compliant - we don't want to force developers to format it though.